### PR TITLE
fix: label transform do not use undefined bitmap

### PR DESCRIPTION
**Pull request by chanwutk**
Fix of [this issue](https://github.com/vega/vega-label/issues/21) from Vega-Label repository.
Prevent labeling algorithm from using `bitmap1` if it is not defined.

### CODE REVIEW

**Comment by domoritz** [comment]
Rebased to master

File: packages/vega-label/src/util/placeAreaLabel.js [hunk 1]
```diff
@@ -1,138 +0,0 @@
-import {textMetrics} from 'vega-scenegraph';
```
Comment (domoritz) [inline packages/vega-label/src/util/placeAreaLabel.js:1]: Did I delete this file when I rebased or did you remove it @chanwutk?

Comment (chanwutk) [reply packages/vega-label/src/util/placeAreaLabel.js:1]: I will check.

Comment (chanwutk) [reply packages/vega-label/src/util/placeAreaLabel.js:1]: I deleted it. The file is not used anywhere.

**Pull request merged**